mbrl/planning/trajectory_opt.py

```diff
@@ -486,6 +489,33 @@ def optimize(
 
         return mu if self.return_mean_elites else best_solution
 
+class ActorCriticOPtimizer(Optimizer):
```

Comment:
This module is meant for algorithms that perform trajectory optimization by simulating trajectories from the dynamics model, which is not the case for Dreamer. In this case, it would be more appropriate to create a new agent class (e.g., `DreamerAgent` or `ActorCriticAgent`) and save it in its own python file. You can follow the example of the `SACAgent` used by MBPO, defined in [this](https://github.com/facebookresearch/mbrl-lib/blob/main/mbrl/planning/sac_wrapper.py) file.

Reply:
@luisenp thanks for pointing it out! I'll make the necessary changes.